code/popup.html

```diff
@@ -123,12 +123,20 @@ <h1 style="text-align: center;">World's first Offline Search Engine</h1>
 
   <div id="help_popup">
       <div id="popup">
-      <form action="#" id="form" method="post" name="form">
-      <img id="close" src="image/12.png" >
-      <h2>Welcome to Quark!</h2>
-      <hr>
-      <h6>This is the World's first offline code Search Engine presented by <a target="_blank" title="Works only when you are online" href="https://github.com/OpenGenus/">OpenGenus</a>. <br> <br> Now search code for any algorithm or data-structure in your favorite language even when you are not connected to the internet! <br> <br> Add frequently visited codes easily to your favorites by clicking on the star icon. <br> <br> Have any suggestions? Feel free to share your valuable feedback at this <a target="_blank" title="Works only when you are online" href="https://discourse.opengenus.org/t/feedback-for-quark/125">Discourse topic</a>!  <br> <br> </h6>
-      </form>
+        <form action="#" id="form" method="post" name="form">
+          <img id="close" src="image/12.png" >
+          <h2>Welcome to Quark!</h2>
+          <hr>
+          <h6>This is the World's first offline code Search Engine presented by <a target="_blank" title="Works only when you are online" href="https://github.com/OpenGenus/">OpenGenus</a>. <br> <br> Now search code for any algorithm or data-structure in your favorite language even when you are not connected to the internet! <br> <br> Add frequently visited codes easily to your favorites by clicking on the star icon. <br> <br> Have any suggestions? Feel free to share your valuable feedback at this <a target="_blank" title="Works only when you are online" href="https://discourse.opengenus.org/t/feedback-for-quark/125">Discourse topic</a>!  <br> <br> </h6>
+          <hr>
+          <div id="share-footer" >   
+            <p align="center" style="padding-top: 8px; margin-bottom: 2px"><b> Share Quark </b></p>
+            <a class="btn btn-default" target= "_blank" onclick="openWindow(600, 900, 'http://www.facebook.com/sharer.php?u=https://chrome.google.com/webstore/detail/opengenus-offline-search/lfoloadpfjildomeafpdopahkdaoofbn');" href="#"><img id="share-buttons" src="image/Facebook.png" alt="Facebook" ></img></a>
```

Comment:
We cannot use `onclick` in a chrome extension. For this to work, one needs to trigger the function with the element on a click event. You may do this or simply, direct it to the share URL which opens on a new tab. Also, include the sharing message.